auto_install/install.sh

```diff
@@ -2225,31 +2381,29 @@ confWireGuard(){
 
 	echo "::: Server Keys have been generated."
 
-	if [ "$pivpnenableipv6" == "1" ]; then
-
-	echo "[Interface]
-PrivateKey = $($SUDO cat /etc/wireguard/keys/server_priv)
-Address = ${vpnGw}/${subnetClass},${vpnGwv6}/${subnetClassv6}
-MTU = ${pivpnMTU}
-ListenPort = ${pivpnPORT}" | $SUDO tee /etc/wireguard/wg0.conf &> /dev/null
+	{
+		echo '[Interface]'
+		echo "PrivateKey = $(${SUDO} cat /etc/wireguard/keys/server_priv)"
+		echo -n "Address = ${vpnGw}/${subnetClass}"
 
-	else
+		if [ "$pivpnenableipv6" == "1" ]; then
+			echo ",${vpnGwv6}/${subnetClassv6}"
+		else
+			echo
+		fi
 
-	echo "[Interface]
-PrivateKey = $($SUDO cat /etc/wireguard/keys/server_priv)
-Address = ${vpnGw}/${subnetClass}
-MTU = ${pivpnMTU}
-ListenPort = ${pivpnPORT}" | $SUDO tee /etc/wireguard/wg0.conf &> /dev/null
+		echo "MTU = ${pivpnMTU}"
+		echo "ListenPort = ${pivpnPORT}"
+	} | ${SUDO} tee /etc/wireguard/wg0.conf &> /dev/null
 
-	fi
 	echo "::: Server config generated."
 }
 
 confNetwork(){
 	# Enable forwarding of internet traffic
-	$SUDO sed -i '/net.ipv4.ip_forward=1/s/^#//g' /etc/sysctl.conf
+	$SUDO sed -i 's/^#(net.ipv4.ip_forward)=.*/\1=1/g' /etc/sysctl.conf
 	if [ "$pivpnenableipv6" == "1" ]; then
-		$SUDO sed -i '/net.ipv6.conf.all.forwarding=1/s/^#//g' /etc/sysctl.conf
+		$SUDO sed -i 's/^#(net.ipv6.conf.all.forwarding)=.*/\1=1/g' /etc/sysctl.conf
```

Comment:
```
# sed -i 's/^#(net.ipv4.ip_forward)=.*/\1=1/g' /etc/sysctl.conf
sed: -e expression #1, char 35: invalid reference \1 on `s' command's RHS

# sed -i 's/^#(net.ipv6.conf.all.forwarding)=.*/\1=1/g' /etc/sysctl.conf
sed: -e expression #1, char 44: invalid reference \1 on `s' command's RHS
```

Reply:
As said [above](https://github.com/pivpn/pivpn/pull/1567#issuecomment-1192800746), on Alpine the file is doesn't have the uncommented sysctl (no kernel parameters file actually, just a comment) and sed does nothing in this case.